FastBank.Domain/TransactionsReport.cs

```diff
@@ -13,5 +13,6 @@ public TransactionsReport(Guid reportId, DateTime createdOn, string pathToFile,
         public DateTime CreatedOn { get; private set; }
         public string PathToFile { get; private set; }
         public User CreatedBy { get; private set; }
+        public int Index { get; set; }
```

Comment:
Write new method SetIndex() to set the index of the report. Make the setter private.